### PR TITLE
Blackshield Rework V1

### PR DESCRIPTION
**Pull request by EliteRiceFarmer**
Following changes were conducted:

Removed uniforms varieties in lockers.
Removed Low-quality patches in accessories closet.

Added two new variety uniform options as boxes.
Added new vendor that can dispense said boxes in return for credits.
Added a Sergeant Major Richard Winters Plushie. (?)

### CODE REVIEW

**Comment by EliteRiceFarmer** [comment]
PS: Mapper, remember to throw the new vendor into the map~

**Comment by r4d6** [comment]
Why use a new vendor and not like, the voucher system?

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 1]
```diff
@@ -218,32 +218,10 @@
...
-	new /obj/item/clothing/under/rank/trooper/green(src)
-	new /obj/item/clothing/under/rank/trooper/cadet(src)
-	new /obj/item/clothing/under/rank/trooper/cadet(src)
-	new /obj/item/clothing/under/rank/trooper/cadet(src)
-	new /obj/item/clothing/under/rank/trooper/cadet(src)
 	new /obj/item/clothing/under/rank/trooper/skirt(src)
 	new /obj/item/clothing/under/rank/trooper/skirt(src)
 	new /obj/item/clothing/under/rank/trooper/skirt(src)
 	new /obj/item/clothing/under/rank/trooper/skirt(src)
-	new /obj/item/clothing/under/rank/fatigues/green(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:237]: Do not remove the boiler suits

File: code/game/machinery/vendor/weapon_vendors.dm [hunk 2]
```diff
@@ -342,6 +342,21 @@
 	product_ads = "Imagine the rare stuff and buy more!;Rare stuff if you buy more!;Buy and get random items!;Don't quit, buy more!;Its not a lootbox, its surprise mechanics!"
 	auto_price = FALSE
 
+/obj/machinery/vending/blackshield_uniform
+	name = "Blackshield Uniform Vendor"
+	desc = "A vendor that can dispense specialized uniform kits for Blackshield Troopers."
+	icon_state = "trashvend"
+	vendor_department = DEPARTMENT_SECURITY
+	products = list(
+					/obj/item/storage/box/blackshield_uniform = 20
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/weapon_vendors.dm:351]:
```suggestion
					/obj/item/storage/box/blackshield_uniform = 20,
```

File: code/game/machinery/vendor/weapon_vendors.dm [hunk 3]
```diff
@@ -342,6 +342,21 @@
...
+					/obj/item/storage/box/blackshield_uniform/green = 20
+					)
+	prices = list(
+					/obj/item/storage/box/blackshield_uniform = 500
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/weapon_vendors.dm:355]:
```suggestion
					/obj/item/storage/box/blackshield_uniform = 500,
```

**Comment by EliteRiceFarmer** [comment]
> Why use a new vendor and not like, the voucher system?

The uniform are supposed to be specialised but not part of the standard package, so they have to buy it.

Just like in real life!

- Me having to pay 800$ for my uniform cause my issued one doesn't fit anymore.

**Comment by cdb-is-not-good** [comment]
When this was discussed the consensus seemed to be to keep the other uniforms in a locker. gating them behind a charge seems like an attempt to curtail their use entirely as you originally suggested.

**Comment by EliteRiceFarmer** [comment]
> When this was discussed the consensus seemed to be to keep the other uniforms in a locker. gating them behind a charge seems like an attempt to curtail their use entirely as you originally suggested. Ditto on /completely/ removing the colored variations of armor, you got told that removing variety from the shield wasn't ideal for a variety of reasons.
45a65cb11510db372e60cfdc29fe011c

**Comment by EliteRiceFarmer** [comment]
This is more problematic than it supposed to be. I will have to take into account the general opinion about Blackshield changes.
No wonder it's a mess considering how everyone is afraid of it becoming MILSIM..

I will write a more thoroughly suggestion. This time on github later. So we can clear all the misinformation,
right now it's just both parties attempting to enforce their stuff onto each other. Including myself included cause most people is misinterpreting the idea.

**Pull request closed without merging**